src/OpenLoco/src/Paint/PaintBridge.cpp

```diff
@@ -688,6 +688,14 @@ namespace OpenLoco::Paint
         },
     };
 
+    constexpr auto kTwosToOnesCompliment = std::to_array<TilePos2>(
```

Comment:
this kind of looks like the rotation calculation is 180 degrees off or something (or was this table part of vanilla?)